app/dashboard/views.py

```diff
@@ -1145,6 +1164,77 @@ def social_contribution_modal(request):
     return TemplateResponse(request, 'social_contribution_modal.html', params)
 
 
+@csrf_exempt
```

Comment:
should we add the `@staff_member_required` decorator here to restrict access?

Reply:
humm can be, Im restricting it using request.user.is_staff, seems to me there are plans to do this a feature for users so not sure if worst changing it.